packages/create-commandkit/tsup.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
```

Comment:
We could share tsup config but it doesn't matter for this pr

Reply:
✅ Would love to have this convenience in the future